content/src/main/content/jcr_root/etc/clientlibs/acs-commons/vendor/qrcode/.content.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[cq.authoring.dialog]"/>
```

Comment:
Same, this shouldnt get injected automatically... would be better to add this to that `acs-commons.qr-code-generator` clientlib cat, and then per documentation (or if you want to get super clever, you could have the NG app create/destroy the following clientlib folder)

```
categories="cq.authoring.editor"
embed="acs-commons.cq-code-generator"
```